src/Testing/AggregateRootTestCase.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EventSauce\LaravelEventSauce\Testing;
+
+use EventSauce\EventSourcing\AggregateRootId;
+use EventSauce\EventSourcing\ConstructingAggregateRootRepository;
+use EventSauce\EventSourcing\DefaultHeadersDecorator;
+use EventSauce\EventSourcing\InMemoryMessageRepository;
+use EventSauce\EventSourcing\MessageDecorator;
+use EventSauce\EventSourcing\MessageDecoratorChain;
+use EventSauce\EventSourcing\MessageDispatcher;
+use EventSauce\EventSourcing\MessageRepository;
+use EventSauce\EventSourcing\PointInTime;
+use EventSauce\EventSourcing\SynchronousMessageDispatcher;
+use EventSauce\EventSourcing\TestUtilities\ConsumerThatSerializesMessages;
+use EventSauce\EventSourcing\Time\Clock;
+use EventSauce\EventSourcing\Time\TestClock;
+use EventSauce\LaravelEventSauce\AggregateRootRepository;
+use EventSauce\LaravelEventSauce\Consumer;
+use Exception;
+use function get_class;
+use LogicException;
+use function method_exists;
+use function sprintf;
+use Tests\TestCase;
```

Comment:
This won't be possible because it falls outside the publishable package scope. Tests aren't shipped. Together with the face that we can't control this namespace because it lives inside the user's app, I believe it won't be possible to tackel this.